### ISSUE TITLE
Refresh Configuration fails on (first) new panel

### ISSUE THREAD

**Issue by justin-radionix** [ORBNETSYS/BGSeries-Public#1]
I have added a new panel under ORBNET Plugins -> BG Series Panels and have entered my panel's IP address, port number and password. When I click Refresh Configuration, I get the pop-up messagebox asking if I'm sure I want to do this. I click Yes and then I get no further updates to indicate it is working or doing anything. If I click on some other item in the Site Navigation tree to leave that page, I get a pop-up that asks if I want to save my changes. I click Yes and then I get a pop-up with the error seen in this screenshot:
https://i.imgur.com/QwaFR2r.png

(side note, it failed every time I tried to attach that screenshot so that's why I'm just entering the url. It's probably a Bosch IT rule blocking it)

I found a log file at C:\ProgramData\Milestone\XProtect Event Server \logs\MIPLogs\MIP2021-12-13.log and if I watch that log in real time using BareTail I don't see any update to the log when I click the Refresh Configuration button. If I leave it to sit for a few minutes, I still don't see any update. It's only when I try to click off the BG Panel page and am prompted to save my changes that the log shows an attempt to connect to the panel and then a failure due to the connection being closed before AuthenticateAsClient can complete.

If I open Wireshark and get it to start capturing all communication with my panel's IP address, I see the same thing - no attempt to connect to the panel until I attempt to leave the page and say Yes to save changes.

So, it seems like there is something hanging and preventing the Refresh Configuration from ever actually starting.

**Comment by EricBourque** [comment]
You must save Milestone Configuration to start the process. Please try to click the refresh configuration button and click on the save button (blue disk) on the top-left of the screen after closing the message box. I will add some more text to make that clear.

**Comment by justin-radionix** [comment]
At first this didn't help any, it would just make the error pop up as soon as I press the save button. But I got to wondering if maybe it was a panel problem because the panel was closing the connection right after the Client Hello portion of the TLS handshake. So I switched to a different panel IP address and it did refresh that one. Then I switched back to the original panel and now it is working too.
So, end result is I can no longer duplicate this problem so we should probably just close it and if I come up with a way to duplicate it again, then we can revisit.
